From 5e73d0f67eb52fd2364442b665c86e0ca84d0894 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 3 May 2024 17:09:54 +0200
Subject: [PATCH 0818/1695] dts: stm32: add a default backup register
 configuration on stm32mp2 boards

The "st,backup-zones" property is mandatory, add a default configuration
at SoC level. It can be superseded by a board configuration.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 46855d4ee9835aca35f49ef050c65b36a3fdd68e dts: stm32: add a default backup register configuration on stm32mp2 boards]
Change-Id: I551dcabfcbcf8930cf6f73f5ecaa3aaa87b8aeb3
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/378282
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/378784
---
 core/arch/arm/dts/stm32mp251.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index 069beb91c..6949a6ee0 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -1051,6 +1051,7 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 			ranges;
+			st,backup-zones = <24 24 24 24 12 12 8>;
 			status = "disabled";
 
 			nvram: nvram@46010100 {
-- 
2.47.2

